### PR TITLE
PMM-10779 Add tests to verify Postgresql vacuum monitoring

### PR DESCRIPTION
**Pull request by peterSirotnak**
https://pmm.cd.percona.com/job/pmm2-ui-tests/23575/allure/#suites/4a0f6f6ddf06d39878432040dfa8bead/565442d7d156938d/

### CODE REVIEW

File: tests/dashboards/verifyExperimentalPostgresqlDashboards_test.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    I, experimentalPostgresqlDashboardsPage,
+  }) => {
+    await I.amOnPage(experimentalPostgresqlDashboardsPage.vacuumDashboardPostgres.url);
+    await I.waitForVisible(experimentalPostgresqlDashboardsPage.elements.barValue, 60);
+    const values = await I.grabTextFromAll(experimentalPostgresqlDashboardsPage.elements.barValue);
+
+    values.forEach((value) => {
+      const valueInt = parseInt(value.replace('%', ''), 10);
+
+      assert.ok(valueInt > 0, 'The value for Postgres vacuum is zero, it supposed to be > 0');
```
Comment (vyarosh) [inline tests/dashboards/verifyExperimentalPostgresqlDashboards_test.js:21]: please use "codecept chai" assertions - it has way better logs and reporting, see online doc: https://www.npmjs.com/package/codeceptjs-chai also readme is available in node_modules/codeceptjs-chai
ex: `I.assertAbove(2, 1, 'Target data not above the given value');` should be great here

File: tests/pages/experimentalPostgresqlDashboards.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      || table.includes('language')
+      || table.includes('rental')
+      || table.includes('staff')
+      || table.includes('payment')
+      ) {
+        await I.verifyCommand(`sudo docker exec pgsql_vacuum_db psql -U postgres -d dvdrental -c 'VACUUM  ( ANALYZE ) ${table.trim()}'`);
+      }
+    }
+  },
+  async waitForLastValues(element, timeoutInSeconds) {
```
Comment (vyarosh) [inline tests/pages/experimentalPostgresqlDashboards.js:38]: please use existing fluent wait method to wrap your condition: https://github.com/percona/pmm-ui-tests/blob/main/tests/custom_steps.js#L68

**Comment by peterSirotnak** [comment]
https://pmm.cd.percona.com/job/pmm2-ui-tests/23766/allure/#suites/4a0f6f6ddf06d39878432040dfa8bead/fb7e8bad965809e0/ Latest test run

File: tests/pages/experimentalPostgresqlDashboards.js (hunk 2 above)
Comment (peterSirotnak) [reply tests/pages/experimentalPostgresqlDashboards.js]: Good idea, done.

File: tests/dashboards/verifyExperimentalPostgresqlDashboards_test.js (hunk 1 above)
Comment (peterSirotnak) [reply tests/dashboards/verifyExperimentalPostgresqlDashboards_test.js]: Good idea, done.

**Review by vyarosh — APPROVED** [review]
Looks good to me! Good job!

**Comment by peterSirotnak** [comment]
> Looks good to me! Good job!

Thank you! :-)

**Pull request merged**